rcl/CMakeLists.txt

```diff
@@ -12,9 +12,12 @@ find_package(rosidl_generator_c REQUIRED)
 
 include_directories(include)
 
+set(CMAKE_C_STANDARD 99)
+set(CMAKE_CXX_STANDARD 14)
```

Comment:
These variables should only be set if they are not yet defined in order to not prevent overriding the value.

Same below.